packages/apollo-server-core/src/plugin/landingPage/default/index.ts

```diff
@@ -203,9 +209,14 @@ curl --request POST \\
   --url '<script>document.write(window.location.href)</script>' \\
```

Comment:
Just to check: the `react-root` and the `fallback` is relevant to all three modes?

Reply:
so I found out that `react-root` is being used in the fallback, and I believe that makes the fallback not show up on the landing page b/c we attach our web app to `react-root` in `studio-landing-page`.

https://github.com/apollographql/studio-landing-page/search?q=react-root

The fallback is relevant for all three modes, because if you can't grab the files from the cdn, the fallback will show. For the new embedded modes (explorer & sandbox) I render them on top of the fallback, so you don't see it if you can successfully grab the files from the cdn.

Reply:
ok so the react-root id is only relevant to one but the div itself is relevant to both (and is blocked/replaced differently across them both but works either way)? sounds good.